apps/prairielearn/python/zygote.py

```diff
@@ -36,7 +36,7 @@
 
 saved_path = copy.copy(sys.path)
 
-drop_privileges = os.environ.get("DROP_PRIVILEGES", False)
+drop_privileges = int(os.environ.get("DROP_PRIVILEGES", "0")) == 1
```

Comment:
Please double check with Nate this won't cause any issues. Generally want to be very careful with changes to this file.

Reply:
Why this change in the first place?

Reply:
Here is the rule: https://docs.astral.sh/ruff/rules/invalid-envvar-default/. My understanding is that this is to ensure that the default and the value are the same type. I.e. you wouldn't want drop_privileges to be `"1" | False`.